### DISCUSSION TITLE
What are custom queries in Flowlauncher settings and how to edit it?

### DISCUSSION THREAD

**Discussion by betamigo98** [Flow-Launcher/Flow.Launcher#1560] [Q&A] [answered]
I understood you can edit any hotkey you want to launch any pluggin, and then even edit hotkeys within a particular pluggin (yt to search in youtube from the web search pluggin for example).

However, I coulnd't understand the keywords and queries in the general setting pane. what is it, what does it do?
See picture below:

![image](https://user-images.githubusercontent.com/105124171/202859443-9b56067d-26cf-474a-b652-4872bb056f22.png)

So in what language should I wrote the query on the right hand of the table and what does it refer to?

Some more context: I would like to perform more than one websearch at the time so my idea was to edit a batch command like : 
chrome.exe google.com/search?q={query} facebook.com/search?q={query}

Thanks for your help!

**Comment by VictoriousRaptor** [comment, marked as answer]
We use word "query" to describe texts to search in Flow Launcher. Click add, input something and click preview and you will understand better.

**Reply by betamigo98** [reply]
Ok thank you I'll try that
